classes/runner.php

```diff
@@ -844,6 +844,23 @@ public function addExtension(extension $extension, extension\configuration $conf
 		return $this;
 	}
 
+	public function getExtension($classname)
+	{
+		$extensions = array();
+		foreach ($this->getExtensions() as $extension) {
+			if (get_class($extension) != $classname) {
+				continue;
+			}
+			$extensions[] = $extension;
+		}
+
+		if (1 != count($extensions)) {
+			throw new \Exception(sprintf("Extension '%s' Not found", $classname));
```

Comment:
@agallou use a more specific exception, [https://github.com/atoum/atoum/blob/master/classes/exceptions/logic/invalidArgument.php] for example ;)